src/set_theory/game/nim.lean

```diff
@@ -259,14 +258,12 @@ by simp
   grundy_value G = grundy_value H ↔ G ≈ H :=
 (grundy_value_eq_iff_equiv_nim _ _).trans (equiv_congr_left.1 (equiv_nim_grundy_value H) _).symm
 
-lemma grundy_value_zero : grundy_value 0 = 0 :=
-by simp
+@[simp] lemma grundy_value_zero : grundy_value 0 = 0 := by simp [equiv_symm nim.nim_zero_equiv]
```

Comment:
Did simp become less powerful here? Do you know why?

Reply:
Because the previous theorem `0 ≈ nim 0` was `simp`, and `simp` can't invert symmetric relations all willy-nilly. I do still think this variable reordering is a net gain, though.